nkms/characters.py

```diff
@@ -158,19 +158,21 @@ def find_best_ursula(self):
         # TODO: Right now this just finds the nearest node and returns its ip and port.  Make it do something useful.
         return self.server.bootstrappableNeighbors()[0]
 
-    def generate_re_encryption_keys(self,
-                                    bob,
-                                    m,
-                                    n):
-        # TODO: Make this actually work.
-        pubkey_enc_bob = bob.seal  # ???  We need Bob's enc key, not sig.
-        kfrags = [
-            'sfasdfsd9',
-            'dfasd09fi',
-            'sdfksd3f9',
-        ]
-
-        return kfrags
+    def generate_rekey_frags(self, alice_privkey, bob_pubkey, m, n):
```

Comment:
I like the proper name usage here - if we *know* that something is only the job of an alice or bob...

include their names in the blob?

(I'll see myself out).